docs/src/tutorials/constraints.md

```diff
@@ -65,7 +65,7 @@ function norm_loss_function(phi, θ, p)
 end
 
 discretization = PhysicsInformedNN(chain,
-                                   GridTraining(dx),
+                                   QuadratureTraining(),
```

Comment:
dx is still defined.

Reply:
same as dt from the other file, it seems to be used in other places in this file, thats why I didnt remove it

Reply:
But none of this file should be using it anymore?

Reply:
Ok I removed the instances of dx but it seems like the program is still creating a grid of some kind (see line 61 and below), maybe only for evaluation. Do I want to delete all that stuff?